sample-kotlin/build.gradle

```diff
@@ -35,7 +35,9 @@ dependencies {
     compile (project(':library')) {
         exclude module: "support-v13"
     }
-    compile project(':processor')
+
+    provided project(':processor')
 
     kapt project(':processor')
```

Comment:
No. @horse315 you are right.

https://github.com/hotchemi/PermissionsDispatcher/blob/master/sample/build.gradle#L27

We had provided dependency for Java example. And if we left only provided dependency, kapt won't work. Don't know how to fix this problem.